sources/packages/backend/apps/api/src/services/notification/gc-notify-actions.service.ts

```diff
@@ -1,4 +1,5 @@
 import { Injectable } from "@nestjs/common";
+import { MessageType } from "@sims/sims-db";
 import { getExtendedDateFormat, getPSTPDTDateTime } from "../../utilities";
 import {
```

Comment:
Not a part of this PR. The name of the file gc-notify-actions.service.ts is not matching with what it does. This service seems to be responsible for student file upload notifications. 
@andrewsignori-aot @guru-aot @ann-aot @andrepestana-aot ?

Reply:
It is right now only dealing with file uploads because it is the only notifications that we have right now. The idea is add the others here also. Not part of this PR but we can drop the `gc` prefix.

Reply:
I didn't understand @dheepak-aot . are you saying this file is taking care of both UI notification and email notification

Reply:
I do agree when we have other services, we can remove the gc prefix.